### PR TITLE
bugfix/#5603

### PR DESCRIPTION
**Pull request by KKatiaa**
#5603 The 'Saved' option is not displayed in the 'Status' filter
#5634 The [object.Object] text is displayed in the 'City' field in the 'Create card' form.
#5575 Field 'Basic cost' don`t saved with number 0,01 in 'Service'
#5574 Field 'Basic cost' don`t saved with number 0,1 in 'Service'

### CODE REVIEW

**Review by atsa21 — APPROVED** [review]

**Review by NadiiaStoiko — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.html [hunk 1]
```diff
@@ -84,7 +84,6 @@
           formControlName="city"
           [placeholder]="cityPlaceholder"
           [matAutocomplete]="autoCity"
-          [attr.disabled]="CardForm.controls['regionNameUk'].dirty ? null : true"
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.html:87]: Remove attr.disabled

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class UbsAdminTariffsCardPopUpComponent implements OnInit, OnDestroy {
     station: ['', Validators.required],
     regionNameUk: ['', Validators.required],
     regionNameEng: [''],
-    city: [{ value: '' }, [Validators.maxLength(40), Validators.required]]
+    city: [{ value: '', disabled: false }, [Validators.maxLength(40), Validators.required]]
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts]: Change city: **disabled: true**

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.html (hunk 1 above)
Comment (KKatiaa) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.html:87]: it is already removed

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts (hunk 2 above)
Comment (KKatiaa) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts]: changed it

**Review by hnativlyubomyr — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Pull request merged**